partiql-parser/fuzz/inputs/seed/query_complex.partiql

```diff
@@ -0,0 +1,12 @@
+SELECT (
```

Comment:
I'm curious is there any requirement for what type of statement could be a seed? Maybe in the future, we could consider using the tests from the conformance test suite as the seeds.

Reply:
Indeed the conformance test suite would be better than these hand rolled queries to seed the corpus. The conformance suite `pass` tests should grow to much better model the universe of features, but also the `failure` queries are great seeds for the fuzzing as well. 

There simply weren't enough of them yet for me to use them, and I'll need to write a different generator to dump just the statements (or perhaps query the files w/ partiql?).